packages/examples/packages/cronjobs/snap.manifest.json

```diff
@@ -17,6 +17,10 @@
     }
   },
   "initialPermissions": {
+    "endowment:rpc": {
+      "dapps": true,
+      "snaps": false
+    },
     "endowment:cronjob": {
       "jobs": [
```

Comment:
Is it allowed to use an empty `jobs` array? I'm thinking if a Snap only wants to use the schedule RPC methods, we should allow that if it isn't already